modules/virt-hot-plugging-cpu.adoc

```diff
@@ -19,3 +19,12 @@ You can increase or decrease the number of CPU sockets allocated to a virtual ma
 +
 If the VM is migratable, a live migration is triggered. If not, or if the changes cannot be live-updated, a `RestartRequired` condition is added to the VM.
 
+[NOTE]
+====
+If a VM has the `spec.template.spec.domain.devices.networkInterfaceMultiQueue` field enabled and CPUs are hot-plugged, the following behavior occurs:
+
+
+* Existing network interfaces that you attach before the CPU hot plug retain their original queue count, even after you add more virtual CPUs (vCPUs). The underlying virtualization technology causes this expected behavior.  
+* To update the queue count of existing interfaces to match the new vCPU configuration, you can restart the VM. A restart is only necessary if the update improves performance.  
+* New VirtIO network interfaces that you hot-plug after the CPU hot plug automatically receive a queue count that matches the updated vCPU configuration.
```

Comment:
```suggestion
* New VirtIO network interfaces that you hot plugged after the CPU hotplug automatically receive a queue count that matches the updated vCPU configuration.
```

Reply:
Verb form is two words. See https://redhat-documentation.github.io/supplementary-style-guide/#hot-plug